doc/releases/changelog-dev.md

```diff
@@ -96,27 +95,39 @@
 
   ```
 
-<h4>Make use of more methods to map from molecules 🗺️</h4>
-
+<h4>More methods to map molecules to qubits 🗺️</h4>
```

Comment:
Very minor, but I had a soft spot for the alliteration and use of a very to start each section.